src/devices/bus/isa/mc1502_fdc.cpp

```diff
@@ -2,7 +2,7 @@
 // copyright-holders:Sergey Svishchev
 /**********************************************************************
 
-    Electronika MC 1502 FDC device
+	Electronika MC 1502 FDC device
```

Comment:
Changes like this are going to create diff churn at srcclean time.  Inside a comment, srcclean replaces tabs with spaced beyond the indentation level of the opening line of the coment, e.g. like this:

```
/*
    unindented comment
*/

\t/*
\t    indented comment
\t*/
```

Is it possible to adjust your settings to not mess with spacing inside comments?